src/main/java/com/uber/kafkaSpraynozzle/KafkaArgs.java

```diff
@@ -21,4 +21,6 @@
     @Option(shortName="w", defaultToNull=true, description="The classpath to use for finding the above-mentioned stats class (default: none)") String getStatsClasspath();
     @Option(shortName="d", defaultToNull=true, description="The arguments for the above-mentioned stats class (default: none)") String getStatsClassArgs();
     @Option(shortName="h", helpRequest=true, description="Display this Help message") boolean getHelp();
+    @Option(shortName="i", defaultValue = "1", description="Number of messages to pack as a batch before POST to endpoint, 1 or 0 to non-packing") int getBatchSize();
+    @Option(shortName="e", defaultToNull=true, description="Enable Least-Response-Time load balancing when posting") String getEnableBalancing();
```

Comment:
The getBoolean always confuses me in that library.
Let me verify locally with my test app first.